test/integration/upload_access_test.rb

```diff
@@ -47,27 +43,22 @@ def assert_sent_private_upload(_upload, content_type)
     host! asset_host
   end
 
-  test 'allows everyone access to general uploads' do
+  test 'allows everyone to be redirected to general uploads' do
```

Comment:
I think this might be better named as something like "redirects all non-attachment, non-hmrc asset requests to the asset host"